cmd/collector/app/builder/span_handler_builder.go

```diff
@@ -45,18 +45,18 @@ var (
 
 // SpanHandlerBuilder builds span (Jaeger and zipkin) handlers
 type SpanHandlerBuilder interface {
-	BuildHandlers() (app.ZipkinSpansHandler, app.JaegerBatchesHandler, error)
+	BuildHandlers(cOps *CollectorOptions) (app.ZipkinSpansHandler, app.JaegerBatchesHandler, error)
 }
 
 // NewSpanHandlerBuilder returns a span handler
-func NewSpanHandlerBuilder(opts ...basicB.Option) (SpanHandlerBuilder, error) {
+func NewSpanHandlerBuilder(flagOpts *flags.Flags, opts ...basicB.Option) (SpanHandlerBuilder, error) {
```

Comment:
`flags.Flags` is not go flags... It's a data structure from `cmd/flags' before there were several global variables which I put into `Flags` struct. Agree that the name is not idea.

Struct contains: 
```go
// Flags holds flags configuration
type Flags struct {
	// Logging defines common setting for controlling logs
	Logging logging
	// RuntimeMetricsFrequency specifies how often Go runtime stats should be reported.
	RuntimeMetricsFrequency time.Duration
	// SpanStorage defines common settings for Span Storage.
	SpanStorage spanStorage
	// DependencyStorage defines common settings for Dependency Storage.
	DependencyStorage dependencyStorage
}
```
Do you have any name proposals? In other packages similar structs are named `*Options*`. We could name it `Options`, `StorageOptions` ?